ocs_ci/framework/conf/default_config.yaml

```diff
@@ -47,5 +47,5 @@ ENV_DATA:
   # Do not change to specific version like v14.2.1-20190430 if not needed
   # cause we don't need to update it each time new 14.x version is released
   # but only once when move to new version like v15.
-  ceph_image: 'ceph/daemon-base:latest-nautilus-devel'
+  ceph_image: 'ceph/ceph:v14.2.2-20190722'
```

Comment:
The commit message and changes dont match, Please add a new commit for this change and also change PR subject to reflect the same (better yet to have this commit as a different PR - not sure why it has to be part of PVC delete)

Reply:
This change was already merged here - https://github.com/red-hat-storage/ocs-ci/pull/466
I'll rebase